### ISSUE TITLE
Cannot get status to change

### ISSUE THREAD

**Issue by jacobwtyler** [nitaybz/homebridge-people#1]
Hi, I've followed the setup instructions and I've opened the ports. I'm using locative to Post to my Dynamic DNC host name. I get a return saying status changed True on my homebridge log and back from locative but for some reason it just won't change the status of the person that I've set it to change. Any suggestions?

**Comment by jacobwtyler** [comment]
Also, you mention ports 51828 and 51829 and different points in your readme file. I've tried both with no success and I've even set a webhook in the config.json file for 51829 and confirmed that the port is being forwarded to my Raspberry Pi with no success. Just wanted to confirm that I should only be using a single port number in both locative Post addresses. Also wanted to confirm I should be using "post" and not "get". I know it is reaching some endpoint because it returns a status change True on a web browser when I point it to the URL you suggested for arrival and departure. However, no matter if I use the False URL or the True URL, I still get a "True" status change message with no change in my actual HomeKit app.

**Comment by andyrooo82** [comment]
i use 51829 in both my config and the post (use post not get)
i had the same issue where the sensor wasn't updated. turned out it was because i had square brackets around my name https://github.com/PeteLawrence/homebridge-people/issues/32.
take these out and it should work fine

**Comment by nitaybz** [comment]
hey Guys, to be honest I created this plugin in order to use only the geofence part of the homebridge-people plugin.
since I created this plugin, homebridge-people made a big progress and have the ability to enable only geofence, or even to rely on wifi just as a fallback.

I would definitely suggest you to go there... and if you insist, I hope @andyrooo82 solution will solve your issue.

Let me know if you need anything else.

**Comment by jacobwtyler** [comment]
wow, I would have NEVER figured that out but thank you andyroo82.  I took the brackets off my name in both URL's and that fixed it!  On a side note, I didn't know that the Homebridge-People plugin had become the better plugin.  I will try using both for now and see if one works better than the other.  For months now, I've been using a combination of Life360 and iConnect Hue to trigger a Hue lamp to four different colors to indicate the first person arriving home and the last person leaving home in order to keep from turning out all the lights and A/C on my wife.  These apps were spotty at best.  Hopefully this will be a more permanent solution and I can actually use my Hue lamp somewhere useful!  Thanks for all the hard work and thanks for the help!

**Comment by andyrooo82** [comment]
Yeah I might go back to using homebridge-people now it includes geofence. I had massive issues when the lights would come on in the middle of the night because our phones dropped off the network. Now it includes geofence there's no reason not to use it anymore (apart from having to rewrite my automations)

**Comment by nitaybz** [comment]
This plugin is deprecated and is no longer maintained.

If you used this for presence detection via geofence, my newer plugin homebridge-network-presence covers ARP-based local-network presence. There are also various community geofence plugins.

Sorry for the long delay on any open issues, and thank you to everyone who used the plugin or contributed.
